content/getting-started/choosing.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Choosing between Slice and Protobuf
+description: Learn why you need an IDL and which IDL makes the most sense for you.
+---
+
+## What are Slice and Protobuf?
+
+IceRPC is at its core a byte-oriented RPC framework: it allows you to easily send requests that carry bytes and receive
+responses that carry more bytes.
+
+This byte-centric architecture and API is exactly what you want if you are moving bytes around. However, it's not
+particularly convenient if you want to define a typed network API for your RPC services.
+
+In the context of RPCs, it is very common to use an [Interface Definition Language] (IDL) to specify your RPCs. These
+definitions are also known as the "contract" between your client and your server.
+
+A code generator then produces code from these definitions. This generated code:
+
+- provides a typed API in the programming language of your choice
+- implements this API by encoding/decoding typed data (integers, strings, structs, etc.) into and from stream of bytes
+in a well-defined format
+
+With an IDL, you don't need to worry about encoding and decoding data structures into and from a portable binary format:
+it's all handled by the generated code and its supporting libraries.
+
+IceRPC includes full support for two IDLs and their associated encoding format: [Slice] and [Protobuf].
+
+Most applications using IceRPC also use either Slice or Protobuf. Let's figure out which one you should pick!
```

Comment:
I would get rid of this sentence, it sounds like it's just repeating the last sentence.
> IceRPC supports Slice and Protobuf

> Applications using IceRPC use Slice or Protobuf

```suggestion
Let's figure out which one you should pick!
```